### ISSUE TITLE
Insufficient escaping in generated man page

### ISSUE THREAD

**Issue by wferi** [ClusterLabs/libqb#428]
The `...` in the `qbarray.h` example gets into the v2.0.2 man page unescaped, leading to the
```
$ man --warnings ./qbarray.h.3.gz >/dev/null
troff: <standard input>:13: warning: macro '..' not defined
```
and it's indeed missing from the output. In the v2.0.1 man page it's protected as `\&.\&.\&.` (which is arguably too much caution) and appears correctly.

**Comment by chrissie-c** [comment]
Are you sure that 2.0.1 quoted the dots like that? docygen2man in 2.0.1 didn't support @code blocks so that part of the doc wouldn't have been in the man page anyway (and isn't on my checkout of the v2.0.1 tag).

Anyway, simplest solution is to add a format before the ellipsis so that nroff doesn't look for a macro 
https://github.com/ClusterLabs/libqb/pull/431

**Comment by wferi** [comment]
Sorry, I mixed up the version numbers. It's 1.0.5 which employs `\&.\&.\&.` in `qbarray.h.3.gz`.

**Comment by chrissie-c** [comment]
ahh that makes a lot more sense :)

**Comment by chrissie-c** [comment]
Fixed in 47ed440397dee9133c31c4d50f7740ab59c8af63
